.github/workflows/tests.yml

```diff
@@ -5,22 +5,29 @@ on:
   pull_request:
 
 env:
-  POETRY_VERSION: 1.1.7
+  POETRY_VERSION: 1.1.11
 
 jobs:
   int:
     name: Integration Tests
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Run integration tests
-      run: |
-        docker-compose -f ./int/docker-compose.yml run tests
-    - name: Print logs on failure
-      if: failure()
-      run: |
-        docker-compose -f ./int/docker-compose.yml logs
-    - name: Clean up integration tests
-      if: always()
-      run: |
-        docker-compose -f ./int/docker-compose.yml down
+      - uses: actions/checkout@v2
+      - name: Run integration tests - Redis cluster
+        run: |
+          docker network create --subnet=172.28.0.0/24 test_network
+          export REDIS_PASSWORD=test1234
+          export NETWORK_NAME=test_network
+          docker-compose -f ./int/docker-compose.yml run tests-cluster
+      - name: Run integration tests - Redis host
+        run: |
+          docker-compose -f ./int/docker-compose.yml run tests-host
```

Comment:
Based upon the error in the tests, it looks like it may be because the host tests section may not be setup all the way

Reply:
@frostyfrog Can you re-run the test? I am not able to replicate this locally [both via commands and `https://github.com/nektos/act`] and it is passing every time.

Reply:
Just mentioning this here, the error in the tests is the following:
> Run docker-compose -f ./int/docker-compose.yml run tests-host
> The REDIS_PASSWORD variable is not set. Defaulting to a blank string.
> The NETWORK_NAME variable is not set. Defaulting to a blank string.
> Network acapy_default declared as external, but could not be found. Please create the network manually using `docker network create acapy_default` and try again.
> Error: Process completed with exit code 1.

The tests for hosts aren't even running. I believe that the following change would fix the tests:
```suggestion
      - uses: actions/checkout@v2
      - name: Run integration tests - Redis cluster
        run: |
          docker network create --subnet=172.28.0.0/24 test_network
          export REDIS_PASSWORD=test1234
          export NETWORK_NAME=test_network
          docker-compose -f ./int/docker-compose.yml run tests-cluster
      - name: Run integration tests - Redis host
        run: |
          docker network create --subnet=172.28.0.0/24 test_network
          export REDIS_PASSWORD=test1234
          export NETWORK_NAME=test_network
          docker-compose -f ./int/docker-compose.yml run tests-host
```

Reply:
@frostyfrog Thanks, can you run the workflow again?